code/src/pages/product-details/ProductDetailsPage.tsx

```
@@ -8,17 +9,25 @@
 import { ProductDetailsPageParams } from "@/pages/product-details/ProductDetails.types";
 import { productNotFoundRedirectConfig } from "@/pages/product-details/ProductsDetailsPage.constants";
 import ProductDetailsContainer from "@/pages/product-details/components/product-details-container/ProductDetailsContainer";
+import { useUpdateViewedProductsMutation } from "@/store/api/viewHistoryApi";
+import { useUpdateViewedProductsMutation } from "@/store/api/viewHistoryApi";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'useUpdateViewedProductsMutation'. Consider to delete it or rename it.

'useUpdateViewedProductsMutation' is defined here:


(lint/suspicious/noRedeclare)